### PR TITLE
Feature/web/display description

### PR DESCRIPTION
**Pull request by Tan82710**
## Description

Display description of services + changed css of Toolbar

## Related issues
- #46 

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] This change requires a documentation update

## Screens
Description1

Description2

### CODE REVIEW

File: web/src/components/Argument/index.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    }, []);
+
+    const isValueValid = (value: any): boolean => {
+        switch (type) {
+            case "string":
+                if (max != null && value.length > parseInt(max)) {
+                    setErrorMessage('Value is over max length');
+                    return false;
+                }
+                if (min != null && value.length > parseInt(min)) {
```
Comment (lucaslehot) [inline web/src/components/Argument/index.tsx]: `if (min != null && value.length > parseInt(min)) {`
->
`if (min != null && value.length < parseInt(min)) {`

File: web/src/components/Argument/index.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        return (
+            <>
+                <FormGroup style={{margin: '10px 0'}}>
+                    <FormControlLabel control={<Checkbox checked={inputValue} onChange={(evt) => onChange(evt.target.checked)}/>} label={name} />
+                </FormGroup>
+            </>
+        )
+    }
+
+    if (type == 'number') {
```
Comment (lucaslehot) [inline web/src/components/Argument/index.tsx]: Pour les types string et number vu que les return sont quasiment les mêmes on peut peut être les rassembler
`if (['number', 'string'].includes(type) {`
et juste dans le TextField mettre
`type={type}`

File: web/src/components/Description/index.tsx [hunk 3]
```diff
@@ -1,17 +1,39 @@
...
+interface ResourceSideBarProps {
+  nodes: Node<ResourceNodeData>[],
+}
 
+const Description = (props: ResourceSideBarProps) => {
+  const [expanded, setExpanded] = React.useState<string | false>(false);
+
+  useEffect(() => {
+    props.nodes.forEach((node: Node<ResourceNodeData>) => {
+      if (node.selected === true) {
```
Comment (lucaslehot) [inline web/src/components/Description/index.tsx]: `if (node.selected === true) {`
->
`if (node.selected) {`

File: web/src/components/Argument/index.tsx (hunk 1 above)
Comment (Tan82710) [reply web/src/components/Argument/index.tsx]: updated

File: web/src/components/Argument/index.tsx (hunk 2 above)
Comment (Tan82710) [reply web/src/components/Argument/index.tsx]: updated

File: web/src/components/Description/index.tsx (hunk 3 above)
Comment (Tan82710) [reply web/src/components/Description/index.tsx]: updated

**Review by lucaslehot — APPROVED** [review]

**Pull request merged**